cosmos/io.py

```diff
@@ -207,7 +212,13 @@ def upload_to_cloud_storage(project_dir: str, source_subpath: str = DEFAULT_TARG
     if not dest_target_dir:
         raise CosmosValueError("You're trying to upload artifact files, but the remote target path is not configured.")
 
-    from airflow.io.path import ObjectStoragePath
+    try:
+        from airflow.sdk import ObjectStoragePath
+    except ImportError:
+        try:
+            from airflow.io.path import ObjectStoragePath
+        except ImportError:
+            pass
```

Comment:
Probably we don't need this here again since we already have a top-level import now? Happy for it to be cleaned up in a separate PR if we don't want to make this change in the PR now, after the many attempts already in this PR now.